contracts/ExitFormat.sol

```diff
@@ -311,36 +305,36 @@ library ExitFormat {
 
             // if lengths don't match the arrays are not equal
             switch eq(length, mload(_postBytes))
-                case 1 {
-                    // cb is a circuit breaker in the for loop since there's
-                    //  no said feature for inline assembly loops
-                    // cb = 1 - don't breaker
-                    // cb = 0 - break
-                    let cb := 1
+            case 1 {
+                // cb is a circuit breaker in the for loop since there's
+                //  no said feature for inline assembly loops
+                // cb = 1 - don't breaker
+                // cb = 0 - break
+                let cb := 1
 
-                    let mc := add(_preBytes, 0x20)
-                    let end := add(mc, length)
```

Comment:
I'm guessing we don't have protection in this repo on solidity formatting?

Reply:
NM it looks like these changes got undone. We can iron out that meander by squashing the PR, if you want.

Reply:
We do in fact - see my chain of Apply Formatting commits. My local editor disagreed with the repo's configured formatter `yarn prettier --check *`.